spec/fixtures/files/datahashes/core/collectionobject1.json

```diff
@@ -12,14 +12,11 @@
     "briefDescription": "Photo moto;chief inspector;",
     "distinguishingFeatures": "Verified account.",
     "comment": "Ndoto moto",
-    "computedCurrentLocationLocationLocal": "~Indeterminate Location~",
-    "computedCurrentLocationLocationOffsite": null,
-    "computedCurrentLocationOrganization": null,
     "title": "Si urembo",
     "titleLanguage": "Swahili",
     "titleType": "generic",
-    "titleTranslation": "Alehandro;Pata pesa",
-    "titleTranslationLanguage": "Spanish;Swahili",
+    "titleTranslation": "Alehandro^^Pata pesa",
+    "titleTranslationLanguage": "Spanish^^Swahili",
```

Comment:
These values are in the title translation subgroup, so need the subgroup field value delimiter. Probably makes more sense if you add a second title group. Also, compare with the anthro collectionobject record and JSON, which has 2 titles in it.